### DISCUSSION TITLE
Proposal: support NO_COLOR in addition to FASTLANE_DISABLE_COLORS environment variable

### DISCUSSION THREAD

**Discussion by 0xced** [fastlane/fastlane#17402] [Ideas] [answered]
[NO_COLOR](https://no-color.org) is an informal standard for command-line utilities:
> All command-line software which outputs text with ANSI color added should check for the presence of a NO_COLOR environment variable that, when present (regardless of its value), prevents the addition of ANSI color.

**Comment by rogerluan** [comment]
Hey @0xced 👋 

Although I wouldn't be a user of this setting, I understand some users prefer to not see colored text.
I think it's an addition that clearly offers benefits with no harm 👍

**Comment by rogerluan** [comment, marked as answer]
This is being proposed here: https://github.com/fastlane/fastlane/pull/17611 🎉
